docs/infrastructure/operations.md

```diff
@@ -0,0 +1,30 @@
+# Infrastructure Operations
+
+This page summarises the service level objectives of the Infrastructure Team.
```

Comment:
SLO has the same accronym as Service Level Obligation (A term I have seen used before) which may cause some confusion while it isn't in the document and is just in the PR title, we should avoid using the acronym

Reply:
Ah, I've only encountered SLO in this meaning.